### PR TITLE
Fix minimum version for SingleValueQuery

### PR DESCRIPTION
**Pull request by dnhatn**
The minimum supported version of SingleValueQuery should be updated to 8.11.

### CODE REVIEW

File: x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SingleValueQuery.java [hunk 1]
```diff
@@ -166,7 +165,7 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
 
         @Override
         public TransportVersion getMinimalSupportedVersion() {
-            return EsqlPlugin.TRANSPORT_MINIMUM_VERSION;
+            return TransportVersion.V_8_500_065; // This is 8.11 - the first version of ESQL
```
Comment (nik9000) [inline x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SingleValueQuery.java:168]: That's a wild version number....

**Review by nik9000 — APPROVED** [review]

**Review by costin — APPROVED** [review]
LGTM

File: x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SingleValueQuery.java (hunk 1 above)
Comment (dnhatn) [reply x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SingleValueQuery.java:168]: Yes, that's why I added a comment :).

**Comment by dnhatn** [comment]
@nik9000 @costin Thanks for reviews.

**Pull request merged**